.github/workflows/lint-go.yml

```diff
@@ -16,4 +16,4 @@ jobs:
     uses: kyma-project/eventing-tools/.github/workflows/lint-go-reusable.yml@main
     with:
         go-version: '1.21'
-        lint-config-uri: https://raw.githubusercontent.com/kyma-project/eventing-tools/b740019d37e5b69c1a1383c71bb5fb5a44c6534e/config/lint/.golangci.yaml
+        lint-config-uri: https://raw.githubusercontent.com/kyma-project/eventing-tools/ca5d5cc431d9fb5cd12e7ef4c6c82331f203c630/config/lint/.golangci.yaml
```

Comment:
Can you please add a description to the PR what changes this will entail? Or point to the corresponding PR?